overrides/aboutCertError.xhtml

```diff
@@ -234,12 +307,20 @@
         <!-- The following sections can be unhidden by default by setting the
              "browser.xul.error_pages.expert_bad_cert" pref to true -->
         <div id="technicalContent" collapsed="true">
-          <h2 class="expander" onclick="toggle('technicalContent');" id="technicalContentHeading">&certerror.technical.heading;</h2>
-          <p id="technicalContentText"/>
+          <h2 class="expander" onclick="toggle('technicalContent');" id="technicalContentHeading">
+            <span class="expander-caret"/>
+            &certerror.technical.heading;
+          </h2>
+          <div>
+            <p id="technicalContentText"/>
+          </div>
         </div>
 
         <div id="expertContent" collapsed="true">
-          <h2 class="expander" onclick="toggle('expertContent');" id="expertContentHeading">&certerror.expert.heading;</h2>
+          <h2 class="expander" onclick="toggle('expertContent');" id="expertContentHeading">
+            <span class="expander-caret"/>
+            &certerror.expert.heading;
+          </h2>
           <div>
```

Comment:
Did you consider including the [`certificateErrorText`](https://github.com/sailfishos-mirror/gecko-dev/blob/600df6aa643726626d2eab36bc0f6e34bbe5ac2a/browser/base/content/aboutNetError.xhtml#L205) and [`getFailedCertificatesAsPEMString()`](https://github.com/sailfishos-mirror/gecko-dev/blob/600df6aa643726626d2eab36bc0f6e34bbe5ac2a/browser/base/content/aboutNetError.js#L621) code?

I can see that it might complicate things, but if it's an easy addition it has some value, I think.

Reply:
getFailedCertificatesAsPEMString() I think had some dependencies that weren't working. And then the button text is from yet other dtd file, and it has to fit into the different page layout.

If we're going to go down the road of taking more and more from aboutNetError.xhtml we should just take the whole thing.

Reply:
Yes, I can see there being layout issues, and it sounds like it's more trouble than it's worth.

I also agree we probably should take the whole thing, but also appreciate that's a much bigger task.